src/screens/GenderSelect.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  Pressable,
+  TouchableOpacity,
+} from 'react-native';
+
+type RadioButtonProps = {
+  gender: string;
+  checkedGender: string;
+};
+
+const GenderSelect: React.FC = ({ navigation }) => {
+  const [checkedGender, setCheckedGender] = useState('');
```

Comment:
初期値で空の文字を入れるのはよくないかな！下みたいな書き方をするといいかも
```suggestion
  const [checkedGender, setCheckedGender] = useState<Gender>(Gender.Male);
```